libs/ledger-live-common/src/exchange/swap/hooks/useProviderRates.test.ts

```diff
@@ -330,6 +333,34 @@ describe("useProviderRates", () => {
     expect(result.current.rates.value).toBeUndefined();
   });
 
+  it("SwapExchangeRateAmountTooLowOrTooHigh should take precedence over SwapExchangeRateAmountTooLow", async () => {
```

Comment:
Where does the `AmountTooHigh` error fit into this? Should it be incorporated into this test or maybe have a separate one?